.github/workflows/presubmit.yml

```diff
@@ -62,6 +62,7 @@ jobs:
         shell: bash
         run: |
           set -x
+          echo "ANDROID_NDK_HOME=$ANDROID_NDK_LATEST_HOME" >> $GITHUB_ENV
```

Comment:
Please add a comment that we need NDK 23 and that default ANDROID_NDK_HOME is 21.

https://github.com/actions/virtual-environments/blob/4f8049546ee84b94b4fe947bab7fa9558b34b195/images/linux/Ubuntu2004-README.md#environment-variables-3

Reply:
Done.